lesson-2/README.md

```diff
@@ -0,0 +1,23 @@
+# Unit 7 Lesson 2: Middleware and Express
+
+## Key Terms
++ Express
++ Middleware
++ Routing
++ JSON
++ Framework
+
+## Essential Questions
+
++ Why might we want to use the Express framework to create our web applications?
++ What is middleware? Why is it useful?
+
+## Learning Assignments
++ [Express Documentation - Getting Started](https://expressjs.com/en/starter/installing.html)
```

Comment:
The Express docs are super clear. Just reading the Getting Started section is really informative. Glad you added that here.